app/views/editions/secondary_nav_tabs/_tagging.html.erb

```diff
@@ -27,13 +27,19 @@
         heading: heading_browse_pages,
         body: "Not tagged to any browse pages",
         link_text: "Tag to a browse page",
-        link_url: "#",
+        link_url: tagging_mainstream_browse_page_edition_path,
```

Comment:
I think this link should not render to a user without permissions. It's not on this card but was on [this tagging card](https://trello.com/c/D9X36Y7R/744-tagging-add-and-edit-data-in-set-govuk-breadcrumb-summary-card) but applicable to all.